src/Processors/QtiV2/In/Interactions/ChoiceInteractionMapper.php

```diff
@@ -65,10 +75,7 @@ public function getQuestionType()
 
         // Build validation
         $validationBuilder = new ChoiceInteractionValidationBuilder(
-            $this->responseDeclaration,
-            array_column($options, 'label', 'value'),
-            $maxChoices,
-            $this->outcomeDeclarations
+            $this->responseDeclaration, array_column($options, 'label', 'value'), $maxChoices, $this->outcomeDeclarations
```

Comment:
Formatting: this will break PSR-2. Arguments in a multi-line method call should each be on a separate line, with correct indentation.

Reply:
Done